jsonb/src/main/java/com/inrupt/client/jsonb/JsonbService.java

```diff
@@ -33,20 +33,28 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.lang.reflect.Type;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A {@link JsonService} using the JakartaEE JSON Bind API.
  */
 public class JsonbService implements JsonService {
 
+    static final Logger LOGGER = LoggerFactory.getLogger(JsonbService.class);
+
     private final Jsonb jsonb;
 
     /**
      * Create a JSON-B service.
      */
     public JsonbService() {
         final JsonbConfig config = new JsonbConfig();
-        config.withPropertyNamingStrategy(PropertyNamingStrategy.LOWER_CASE_WITH_DASHES);
+        config.withPropertyNamingStrategy(new LowerCamelCase());
```

Comment:
This is a very general change.
_All_ JSON-B functionality from now on willuse this new naming strategy.

I have no problem with it, except to note that we'll want to communicate in release notes.

Reply:
On second thought, I do see a problem.

JSON-B has its default naming strategy (whatever that is).
Now we make our JSON-B facilities use a different one from the default.
And it's actually one we've written ourselves.
This might confuse developers.

At minimum I'd like to see this prominently reflected in javadocs.
Possibly I'd like us to reconsider this.

Perhaps the use of this naming strategy belongs on a level different from _the_ JSON-B class?